sgx/sgx_mock_test.go

```diff
@@ -0,0 +1,41 @@
+package sgx_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/zama-ai/fhevm-go/fhevm/tfhe"
+	"github.com/zama-ai/fhevm-go/sgx"
+	"pgregory.net/rapid"
+)
+
+var sgxPlaintextDec *rapid.Generator[sgx.SgxPlaintext] = rapid.Custom(func(t *rapid.T) sgx.SgxPlaintext {
```

Comment:
Let's rename to Gen for Generator

```suggestion
var sgxPlaintextGen *rapid.Generator[sgx.SgxPlaintext] = rapid.Custom(func(t *rapid.T) sgx.SgxPlaintext {
```